spec/lib/myfinance/request_spec.rb

```diff
@@ -20,10 +20,37 @@
       subject.run
     end
 
-    it 'generates the base 64 of the user token' do
+    it "generates the base 64 of the user token" do
       allow_any_instance_of(Typhoeus::Request).to receive(:run)
       expect(Base64).to receive(:strict_encode64).with("my-auth-hash:X").and_call_original
       subject.run
     end
+
+    context "when account_id is empty" do
+      subject { described_class.new({ method: 'post', params: {}, url: Myfinance.configuration.url, user_agent: 'My Test User-Agent', token: 'my-auth-hash', account_id: "" }) }
```

Comment:
Line is too long. [176/120]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.